net/openssh/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           compiler_blacklist_versions 1.0
 
 name                openssh
 version             9.4p1
-revision            0
+revision            1
```

Comment:
Please do not increase the revision.

Reply:
My misunderstanding, I thought revisions were to be incremented any time there was a change local to MacPorts which was not congruent with the upstream project's version numbers?

Reply:
Read [the section on the revision keyword in the Guide](https://guide.macports.org/#reference.keywords). It gives examples for when and when not to increase the revision.